tachyon/zk/plonk/permutation/permutation_verifying_key.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TACHYON_ZK_PLONK_PERMUTATION_PERMUTATION_VERIFYING_KEY_H_
+#define TACHYON_ZK_PLONK_PERMUTATION_PERMUTATION_VERIFYING_KEY_H_
+
+#include <utility>
+#include <vector>
+
+#include "tachyon/base/buffer/copyable.h"
+#include "tachyon/zk/plonk/commitment.h"
+
+namespace tachyon {
+namespace zk {
+
+template <typename Curve>
+class PermutationVerifyingKey {
+ public:
+  PermutationVerifyingKey() = default;
+
+  explicit PermutationVerifyingKey(const Commitments<Curve>& commitments)
+      : commitments_(commitments) {}
+
+  explicit PermutationVerifyingKey(Commitments<Curve>&& commitments)
+      : commitments_(std::move(commitments)) {}
```

Comment:
Can. you remove the empty lines?
```suggestion
  PermutationVerifyingKey() = default;
  explicit PermutationVerifyingKey(const Commitments<Curve>& commitments)
      : commitments_(commitments) {}
  explicit PermutationVerifyingKey(Commitments<Curve>&& commitments)
      : commitments_(std::move(commitments)) {}
```